versions/1/data_stream/manifest.spec.yml

```diff
@@ -156,6 +156,12 @@ spec:
           enabled:
             description: Is stream enabled?
             type: boolean
+          multi:
+            description: Can stream be defined multiple times
+            type: boolean
+            default: false
+            examples:
```

Comment:
nit: well, this is a boolean field. I don't think we need to give an example here :)